managed-databases/redis/reference-content/default-user-permissions.mdx

```diff
@@ -0,0 +1,42 @@
+---
+meta:
+  title: Understanding default user permissions
+  description: Learn more about the default user permissions for Manage Database for Redis™
+content:
+  h1: Understanding default user permissions
+  paragraph: Learn more about the default user permissions for Manage Database for Redis™
+tags: databases user redis username password
+categories:
+  - managed-databases
+  - redis
+---
+
+When you create a Redis™ Database Instance, a single default user is a created with the following permissions:
+
+```
++client +cluster|
+info +cluster|
+nodes +cluster|
+myid +cluster|
+slots +cluster|
+shards allkeys
+```
+
+<Message type="important">
+  The default permissions cannot be edited. It is also not possible to create additional users.
+</Message>
+
+You can specify a name for this user when you create a Database Instance via the Scaleway API or the CLI.
```

Comment:
this is also possible in the console. Only the edit of the user is limited to CLI and API

Reply:
```suggestion
You can specify a name for this user when you create a Database Instance.
```